### PR TITLE
v1.1.0

### PR DESCRIPTION
**Pull request by jonato1**
- Feature: Add interoperability for Blueair AWS devices (#24) connecting in the EU region
- Other: Update various dependencies to improve security and reduce vulnerabilities
- Other: Update enableAWS title in config.schema.json to indicate it is required for AWS-enabled devices
- Other: Convert INFO level logging to DEBUG level logging
- Other: Add error handling for new Blueair AWS Environment Variables
- Other: Fix indenting and other formatting issues with npm run lint -- --fix

Hey Fraser - Can you do a quick review and then if you think it looks good, create a pull request from develop to master and release? With all of the various security updates, I just want to make sure it doesn't negatively impact any non-AWS connectivity and functionality. 

Also, I was thinking about creating some wiki pages specific to the AWS stuff. I'm not sure what level of access I need (i.e. collaborator), but when you get a chance, would you mind looking into that?

Thanks,
Jon

### CODE REVIEW

**Review by fjs21 — CHANGES_REQUESTED** [review]
It's very difficult for me to review the changes in the blueair-api.ts file as many sections seem to be changed (but likely aren't). Any chance you could break out AWS support to a new file - let's say blueair-AWSapi.ts?

It would make it much easier to review/approve stuff more quickly.

**Review by fjs21 — COMMENTED** [review]
Otherwise looks good.

File: src/platform.ts [hunk 1]
```diff
@@ -30,12 +32,12 @@ export class BlueAirHomebridgePlatform implements DynamicPlatformPlugin {
     // initializing login information
     this.log = log;
 
-    if(config.username == undefined || config.password == undefined){
+    if(config.username === undefined || config.password === undefined){
       this.log.error('Missing BlueAir API credentials.');
       return;
     }
 
-    this.blueair = new BlueAirApi(this.log, config.username, config.password);
```
Comment (fjs21) [inline src/platform.ts:38]: won't removal of this line prevent the old blueair API from being properly initialized?

Comment (jonato1) [reply src/platform.ts:38]: Very much so! I just had a fun 15 minutes where I was staring at the code trying to figure out why I was getting a TypeError. Latest commit should not only fix that, but finish the refactor.

**Review by fjs21 — APPROVED** [review]

**Pull request merged**